.github/workflows/notebook-test.yml

```diff
@@ -25,52 +25,48 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Get all changed notebooks
-        id: all-changed-notebooks
+      - name: Get relevant changed files
+        id: all-changed
         uses: tj-actions/changed-files@af2816c65436325c50621100d67f6e853cd1b0f1
         with:
-          files: "docs/**/*.ipynb"
+          files: "{docs/**/*.ipynb,scripts/nb-tester/**/*}"
           separator: "\n"
-      - name: Get changed config files
-        id: changed-config
-        uses: tj-actions/changed-files@af2816c65436325c50621100d67f6e853cd1b0f1
-        with:
-          files: "scripts/nb-tester/**/*"
-      - name: Check for notebooks that require Linux
-        id: linux-changed-files
-        uses: tj-actions/changed-files@af2816c65436325c50621100d67f6e853cd1b0f1
-        with:
+          write_output_files: true
```

Comment:
Annoyingly, this always writes to the same files, regardless of the `id`. To work round this, I'm just detecting all relevant changed files in a single step, then filtering paths in the later steps.